layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-bb.org_%.bbappend

```diff
@@ -17,8 +17,13 @@ SRC_URI:append:beaglebone-ai64-k3r5 = " \
   file://0005-env_default.h-Add-support-for-OS_KERNEL_CMDLINE.patch \
 "
 
+SRC_URI:append:beagleplay = " \
```

Comment:
Squash these u-boot changes in the other commit that is adding code related to u-boot

Reply:
OK

Reply:
Squashed commits, as requested.

Reply:
Not resolved